addons/l10n_in_ewaybill/views/l10n_in_ewaybill_views.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_in_ewaybill_form_view" model="ir.ui.view">
+        <field name="name">l10n.in.ewaybill.form.view</field>
+        <field name="model">l10n.in.ewaybill</field>
+        <field name="arch" type="xml">
+            <form string="Ewaybill" create="false">
+                <header>
+                    <button name="action_generate_ewaybill"
+                            string="Generate e-Waybill"
+                            class="oe_highlight"
+                            type="object"
+                            invisible="state != 'pending'"
+                            data-hotkey="g"
+                    />
```

Comment:
https://github.com/odoo/odoo/blob/2d06606ab2bff4ba6647eafc051775fcc97580ed/addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.xml#L18-L26

Nowadays we are more using it like this because it's more flexible to manage attributes